frontend/packages/dev-console/integration-tests/support/pageObjects/quickStarts-po.ts

```diff
@@ -0,0 +1,37 @@
+export const quickStartsPO = {
+  quickStartTitle: '[data-test="page-title"]',
+  filterKeyword: 'input.pf-c-search-input__text-input',
+  statusFilter: 'button.pf-c-select__toggle',
+  statusDropdown: '[aria-label="Select filter"]',
+  statusComplete: '[data-key="Complete"]',
+  emptyState: 'div.pf-c-empty-state__content',
+  clearFilter: '[data-test="clear-filter button"]',
+  cardStatus: '[data-test~="tile"] [data-test~="status"]',
+};
+
+export const quickStartSidebarPO = {
+  quickStartSidebarBody: '[data-test~="drawer"] [data-test~="content"]',
+  startButton: `[data-test="Start button"]`,
+  nextButton: '[data-test="Next button"]',
+  closeButton: '[data-test="Close button"]',
+};
+
+export const quickStartDisplayNameToName = (displayName: string) => {
+  switch (displayName) {
+    case 'Getting started with a sample application': {
+      return 'sample-application';
+    }
+    case 'Install the OpenShift Pipelines Operator': {
+      return 'explore-pipelines';
+    }
+    default: {
+      throw new Error('Option is not available');
+    }
+  }
+};
+
+export function quickStartCard(quickStartDisplayName: string) {
+  const tile = quickStartDisplayNameToName(quickStartDisplayName);
+  const quickStartName = `[data-test~="tile"][data-test~="${tile}"]`;
+  return quickStartName;
+}
```

Comment:
Let us use quickStartName instead of tile. The next line is a `selector`, you can also directly return it or use `selector` here.

```suggestion
export function quickStartCard(quickStartDisplayName: string) {
  const quickStartName = quickStartDisplayNameToName(quickStartDisplayName);
  return `[data-test~="tile"][data-test~="${tile}"]`;
}
```

Reply:
Updated